content/blog/scales-1-2-0/index.Rmd

```diff
@@ -0,0 +1,174 @@
+ggplot2::theme_set(ggplot2::theme_gray(18))
+set.seed(1014)
+```
+
+
+```{=html}
+<!--
+TODO:
+* [x] Look over / edit the post's title in the yaml
+* [x] Edit (or delete) the description; note this appears in the Twitter card
+* [x] Pick category and tags (see existing with `hugodown::tidy_show_meta()`)
+* [ ] Find photo & update yaml metadata
+* [x] Create `thumbnail-sq.jpg`; height and width should be equal
+* [x] Create `thumbnail-wd.jpg`; width should be >5x height
+* [x] `hugodown::use_tidy_thumbnails()`
+* [x] Add intro sentence, e.g. the standard tagline for the package
+* [x] `usethis::use_tidy_thanks()`
+-->
+```
+
+We're very pleased to announce the release of [scales](https://scales.r-lib.org) 1.2.0.
+The scales package provides much of the infrastructure that underlies ggplot2's scales, and using it allow you to customize the transformations, breaks, and labels used by ggplot2.
+You can install it from CRAN with:
+
+```{r, eval = FALSE}
+install.packages("scales")
+```
+
+This blog post will show off a few new features for labeling numbers, log scales, and currencies.
+You can see a full list of changes in the [release notes](https://github.com/r-lib/scales/blob/main/NEWS.md).
+
+```{r setup}
+library(ggplot2)
+library(scales)
+```
+
+## Numbers
+
+`label_number()` is the workhorse that powers ggplot2's formatting of numbers, including `label_dollar()` and `label_comma()`.
+This release added a number of useful new features.
+
+The most important is a new `scale_cut` argument that makes it possible to independently scales different parts of the range.
+This is useful for scales that span multiple orders of magnitude. Take the following two examples which don't get great labels by default:
+
+```{r}
+df1 <- data.frame(
+  x = 10 ^ runif(1000, 2, 9),
+  y = runif(1000)
+)
+df2 <- df1 |> dplyr::filter(x <= 1.25 * 10^6)
+
+plot1 <- ggplot(df1, aes(x, y)) + 
+  geom_point() +
+  labs(x = NULL, y = NULL)
+plot1 + scale_x_log10()
+plot2 <- ggplot(df2, aes(x, y)) + 
+  geom_point()+
+  labs(x = NULL, y = NULL)
+plot2
+```
+
+You can use `cut_short_scale()` to show thousands with a K suffix, millions with a M suffix, and billions with a B suffix:
+
+```{r}
+plot1 +
+  scale_x_log10(
+    labels = label_number(scale_cut = cut_short_scale())
+  )
+plot2 +
+  scale_x_continuous(
+    labels = label_number(scale_cut = cut_short_scale())
+  )
+```
+
+(If your country uses 1 billion to mean 1 million million, then you can use `cut_long_scale()` instead of `cut_short_scale()`.)
+
+You can use `cut_si()` for SI labels:
+
+```{r}
+plot1 + 
+  scale_x_log10(
+    labels = label_number(scale_cut = cut_si("g"))
+  )
+plot2 + 
+  scale_x_continuous(
+    labels = label_number(scale_cut = cut_si("Hz"))
+  )
+```
+
+This replaces `label_number_si()` because it incorrectly used the [short scale abbreviations](https://en.wikipedia.org/wiki/Long_and_short_scales) instead of the correct [SI prefixes](https://en.wikipedia.org/wiki/Metric_prefix).
```

Comment:
```suggestion
This replaces `label_number_si()` because it incorrectly used the [short-scale abbreviations](https://en.wikipedia.org/wiki/Long_and_short_scales) instead of the correct [SI prefixes](https://en.wikipedia.org/wiki/Metric_prefix).
```